testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -1592,7 +1592,8 @@ impl EventDispatcher {
     }
 
     pub fn process_new_attachments(&self, attachments: &[(AttachmentInstance, Attachment)]) {
-        let interested_observers: Vec<_> = self.registered_observers.iter().enumerate().collect();
+        let interested_observers = self.filter_observers(&self.mempool_observers_lookup, true);
```

Comment:
I don't think this is the correct hash set to use for attachment events. None of the others look like a better option either. Maybe we should use `any_event_observers_lookup`, or maybe we should add a new one. I'm not entirely sure.

Reply:
The specific issue (from the sbtc team that is actively using the event system) is about this /attachments/new event. Can we have a new event id dedicated to it?